util/EnKF/gfs/src/adjustps.fd/adjustps.f90

```diff
@@ -32,6 +32,9 @@ program adjustps
 
   real,parameter:: zero=0.0_4, one=1.0_4
 
+! Declare externals
+  external:: w3tagb, vintg, w3tage
+
   character*500 filename_1,filename_2,filename_o
```

Comment:
Why there is some code changes in the porting? Is it to match the new versions of libs?

Reply:
While compiling, several warning messages were generated for the utilities.  While moving from Intel 18 to Intel 19, the functionality of implicit none was changed with the warn all and warn declaration compiler options.  With Intel 18, so long as all variables are explicitly declared, there were no issues.  Now, with Intel 19, external variable declarations are required.  The utilities are compiled with -warn all, so these changes are required to remove all of the warning messages due to non-declared external variables in the utilities.